src/components/CategoryList.js

```diff
@@ -92,7 +83,8 @@ CategoryList.propTypes = {
   data: PropTypes.array,
   noSubtitle: PropTypes.bool,
   refreshControl: PropTypes.object,
-  hasSectionHeader: PropTypes.bool
+  hasSectionHeader: PropTypes.bool,
```

Comment:
if you remove the usage of `hasSectionHeader`, it can be removed here as well, right?

Reply:
I haven't deprecated `hasSectionHeader`, it just gets deleted and re-added when I add a new line.

`hasSectionHeader` is still in use above.

```
renderSectionHeader = ({ section: { title, data } }) => {
    const { hasSectionHeader } = this.props;

    if (!data?.length || !hasSectionHeader) return null;
...
```

Reply:
ah i see